### ISSUE TITLE
refactor - remove initial state userReducer.user and fix StyledLink+gitignore BulmaStyleOverride.css

### ISSUE THREAD

**Issue by joshmorel** [chingu-coders/Voyage2-Bears-04#48]
userReducer.user should be null by default currently is `{ username: 'User' }` - this is causing issues with authentication and rating.

Also StyledLink is causing an issue for authentication because we need an anchor not a react-router-dom "Link"

**Comment by joshmorel** [comment]
resolved by #49

**Comment by eliotn** [comment]
I think we need to add an additional step that recreates the deleted css file for when we run the project.  I am getting errors now when attempting to run the project and the following travis error:
```
The command "yarn test" exited with 1.
5.09s$ yarn build:client
yarn build:client v0.27.5
$ react-scripts build build-css
Creating an optimized production build...
Failed to compile.
Module not found: Error: Can't resolve './BulmaStyleOverrides.css' in '/home/travis/build/chingu-coders/Voyage2-Bears-04/src'
error Command failed with exit code 1.
The command "yarn build:client" exited with 1.
cache.2
store build cache
0.01s
7.19snothing changed, not updating cache
Done. Your build exited with 1.
```

**Comment by joshmorel** [comment]
Yeah maybe... The css thing is kind of completely different than the other issues I am trying to fix. So why don't I get revert that part and leave it for parking lot.

**Comment by joshmorel** [comment]
Also for travis - none of the builds ever pass. We've just kept ignoring it (we have no prod app anyways). I think we either need to stop using travis until we have a real prod env or actually resolve the issues causing it to fail.

**Comment by joshmorel** [comment]
closed by #53
